### PR TITLE
Rockstar Games Launcher v3289815

### PR DESCRIPTION
**Pull request by ArnaudLier**
Added Rockstar Games Launcher v3289815
I don't really know what the version of it.
There are several things that can be the launcher version here, I took the first.
image

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/winget-pkgs/pull/426)

### CODE REVIEW

**Comment by KevinLaMS** [comment]
Installation failed for me.  Are those switches correct?  Installer failed with exit code: 1

**Comment by ZeProf2Code** [comment]
Hmm. That's weird it works fine for me 🤔

**Comment by KevinLaMS** [comment]
@ZeProf2Code 
I was looking at this again.  I was unable to get this to work and the binary changed under the covers.  I have submitted a PR that updates the hash.  it has an issue with smartscreen that i am working through.  Would you mind seeing reviewing the PR and doing the following:
1) sign off on the new PR.
2) close this PR.
https://github.com/microsoft/winget-pkgs/pull/3287

thanks

File: manifests/Rockstar/RockstarGamesLauncher/v3289815.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+License: Copyright (c) Rockstar Games
+LicenseUrl: https://www.rockstargames.com/eula
+AppMoniker: rockstargameslauncher
+Tags: launcher, rockstar games
+Description: The official Rockstar Games Launcher
+Homepage: https://socialclub.rockstargames.com/rockstar-games-launcher
+Installers:
+  - Arch: x64
+    Url: https://gamedownloads.rockstargames.com/public/installer/Rockstar-Games-Launcher.exe
+    Sha256: A61BF1561B39B84E80C87F6616961D0FD22296591345BE07CEEA2F8A89921D3B
```
Comment (mddtest) [inline manifests/Rockstar/RockstarGamesLauncher/v3289815.yaml:14]: InstallerSha256: A3F89DE5B4EF531B270240E872392000B87077637EF4FAE52C945FACB5689E96

File: manifests/Rockstar/RockstarGamesLauncher/v3289815.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+Name: Rockstar Games Launcher
+Publisher: Rockstar Games
+License: Copyright (c) Rockstar Games
+LicenseUrl: https://www.rockstargames.com/eula
+AppMoniker: rockstargameslauncher
+Tags: launcher, rockstar games
+Description: The official Rockstar Games Launcher
+Homepage: https://socialclub.rockstargames.com/rockstar-games-launcher
+Installers:
+  - Arch: x64
```
Comment (mddtest) [inline manifests/Rockstar/RockstarGamesLauncher/v3289815.yaml:12]: - Arch: x86

**Comment by mddtest** [comment]
Still wont do a silent install but updated to laded exe

**Comment by ngbeken1monday** [comment]
I want to learn how to build window with azure

**Comment by denelon** [comment]
@ngbeken1monday try https://learn.microsoft.com/

**Pull request closed without merging**